### PR TITLE
Update Restic documentation for RancherOS

### PR DESCRIPTION
**Pull request by ipochi**
For #1347 

Signed-off-by: Imran Pochi <pochiimran@yahoo.co.in>

### CODE REVIEW

File: docs/restic.md [hunk 1]
```diff
@@ -40,6 +40,21 @@ cross-volume-type data migrations. Stay tuned as this evolves!
...
+   path: /var/lib/kubelet/pods
+   ```
+
+   to 
+
+   ```
+   hostPath:
+   path: /opt/rke/var/lib/kubelet/pods
+   ```
+
```
Comment (kmova) [inline docs/restic.md]: Just wondering if it would be a good idea to add a directory for rancher os, just like aws, azure etc.

Comment (nrb) [reply docs/restic.md]: > just like aws, azure etc.

I'm not sure which directories you're referring to for this. The examples directory?

Comment (kmova) [reply docs/restic.md]: yes @nrb, the examples directory that can includes restic ds YAML with the modified path for rancher os. However a generic instruction with Rancher as example also is fine.

**Comment by ipochi** [comment]
@kmova @nrb , can we agree upon the docs  and close this ?

File: docs/restic.md (hunk 1 above)
Comment (nrb) [reply docs/restic.md]: @kmova I think that would be fine, but I think this can be merged as is, too.

**Pull request merged**